src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -1178,6 +1184,18 @@ public void setMaximumReplicationFactorThreshold (int warn, int fail)
         DEFAULT_CONFIG.setMaximumReplicationFactorThreshold(warn, fail);
     }
 
+    @Override
+    public Map<String, Object> getPasswordValidatorConfig()
+    {
+        return password.getConfig();
+    }
+
+    @Override
+    public void reconfigurePasswordValidator(Map<String, Object> config)
+    {
+        password.reconfigure(config);
```

Comment:
it feels like we need some flag to disable the ability to reconfigure the password validator via JMX?

Reply:
https://github.com/apache/cassandra/pull/3384/commits/ef034a41084200ba3dcdcc7d3682fd65db34abf8